docker-image/entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+cd /workspace
```

Comment:
could we use the environmental variable for this as well?

Reply:
yes, this would then be passed as argument at runtime, and its default value set to /workspace.

Reply:
done